frontend/src/containers/PublicGroup.js

```diff
@@ -248,25 +243,26 @@ export class PublicGroup extends Component {
   // Used after a donation
   refreshData() {
     const {
-      group,
-      fetchGroup,
+      slug,
+      fetchProfile,
       fetchUsers,
       fetchTransactions
     } = this.props;
 
     return Promise.all([
-      fetchGroup(group.id),
-      fetchUsers(group.id),
-      fetchTransactions(group.id, FETCH_DONATIONS_OPTIONS)
+      fetchProfile(slug),
+      fetchTransactions(slug, FETCH_DONATIONS_OPTIONS),
+      fetchTransactions(slug, FETCH_EXPENSES_OPTIONS),
```

Comment:
Good point. I'll take of that in the new PR if you don't mind, because I got rid of `fetchTransactions`